x-pack/plugins/spaces/public/space_selector/space_selector.scss

```diff
@@ -4,6 +4,9 @@
 
 .spcSelectorBackground {
   @include kibanaFullScreenGraphics;
+}
+
+.spcSelectorBackground__nonMixinAttributes {
```

Comment:
Self Review: The mixin was overwriting the z index which caused only the background graphic to show up.

I should also not that the background graphic does not show up on this branch or main, so maybe we could have some design/UX help here